records_mover/db/redshift/loader.py

```diff
@@ -158,6 +157,7 @@ def known_supported_records_formats_for_load(self) -> List[BaseRecordsFormat]:
             }),
             # Supports newlines in strings, but not empty strings.
             DelimitedRecordsFormat(variant='bluelabs'),
+            AvroRecordsFormat(),
```

Comment:
Let's set Avro as the least preferred format for now.  Maybe in the future Records Mover could be smart enough to fix up the columns after loading the file?